instrumentation/src/com/intellij/rt/coverage/instrumentation/TouchCounter.java

```diff
@@ -57,19 +63,19 @@ private void touchBranch(final boolean trueHit, final int jumpIndex, int line) {
     mv.visitVarInsn(Opcodes.ALOAD, getOrCreateLocalVariableIndex());
     InstrumentationUtils.pushInt(mv, line);
     InstrumentationUtils.pushInt(mv, jumpIndex);
-    mv.visitInsn(trueHit ? Opcodes.ICONST_0 : Opcodes.ICONST_1);
+    mv.visitInsn(trueHit ? Opcodes.ICONST_1 : Opcodes.ICONST_0);
```

Comment:
ouch

Reply:
I've inverted it [here](https://github.com/JetBrains/intellij-coverage/blob/172843b2414cdb3764752d8063a3662243aa1716/instrumentation/src/com/intellij/rt/coverage/instrumentation/data/BranchDataContainer.java#L79), so everything works the same.